### DISCUSSION TITLE
Sound doesn't work on IOS only Android

### DISCUSSION THREAD

**Discussion by ramzidaher** [google/model-viewer#4347] [Q&A]
### Description

I'm working with AR models and sounds, currently, when you go into AR mode and place the model, the sound plays only in Android, but on IOS it doesn't, I have tried a bunch of things, but didn't work. Is there a solution for this, I tried some JS doesn't seem to work. By the way, I'm using sound= in model viewer. What's weird is that googles AR 3D animals have sounded on IOS and Android! Note I tired Ogg and mp3 for the audio file.

#### Live Demo

https://spark-model-and-audio.glitch.me

### Version

- model-viewer: 3.1.1

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [X] WebXR
- [X] SceneViewer
- [X] QuickLook

**Comment by elalish** [comment]
iOS has no WebXR support so AR is through QuickLook, which is not the browser, so any browser-based sound or customization is lost. The way the animals did it is to make their own USDZ files equivalent to the GLB that contain their own sound and pass that to `ios-src`. FYI @meshula.

**Comment by ramzidaher** [comment]
Oh okay thank you very much, what is some good tutorials to add audio to USDZ format?
